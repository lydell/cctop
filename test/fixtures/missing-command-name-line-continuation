\ cont
  \ cont

  # comment
	\ cont


# comment
\cont

command
\cont
