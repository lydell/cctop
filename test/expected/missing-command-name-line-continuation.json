{
  "commands": [
    {
      "name": "command",
      "value": "\ncont",
      "lines": [[11, 8, 0], [12, 2, 4]],
      "line": 11,
      "column": 1,
      "endLine": 12,
      "subCommands": []
    }
  ],
  "errors": [
    {
      "id": "missing-command-name-line",
      "line": 1,
      "column": 1,
      "command": {
        "name": null,
        "value": "\ncont\ncont\ncont\ncont",
        "lines": [[null, null, 0], [1, 3, 4], [2, 5, 4], [5, 4, 4], [9, 2, 4]],
        "line": null,
        "column": null,
        "endLine": 9,
        "subCommands": []
      }
    }
  ]
}